item.rb

```diff
@@ -0,0 +1,26 @@
+require 'date'
+
+class Item
+  def initialize(id, **options)
+    @id = id
+    @genre = options[genre]
```

Comment:
I think this is a better place to initialize id, and I prefer to use only one parameter for initializing, maybe initialize(**options) and put id inside options